01_Exercises/Lab 4 to 6/repository.py

```diff
@@ -0,0 +1,138 @@
+class Expense:
```

Comment:
Modulul asta ar trebui sa se cheme `entities` in loc de `repository` pentru ca nu are functie de repository. Contine doar o entitate din programul tau (Expense).